app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -7,11 +7,9 @@ import type { FormStepComponentProps } from '@18f/identity-form-steps';
 import GeneralError from './general-error';
 import TipList from './tip-list';
 import { SelfieCaptureContext } from '../context';
-import {
-  DocumentCaptureSubheaderOne,
-  SelfieCaptureWithHeader,
-  DocumentFrontAndBackCapture,
-} from './documents-and-selfie-step';
+import { DocumentCaptureSubheaderOne } from './documents-and-selfie-step';
+import { DocumentsCaptureStepComponent } from './documents-step';
+import { SelfieStepComponent } from './selfie-step';
```

Comment:
as a naming convention, it doesn't appear we commonly name our components "...Component"

how about something like `SelfieStep` or `SelfieCaptureStep`?